plugins/usi-dock/fu-usi-dock-mcu-device.c

```diff
@@ -718,7 +718,10 @@ fu_usi_dock_mcu_device_cleanup(FuDevice *device,
 	fwupd_request_add_flag(request, FWUPD_REQUEST_FLAG_ALLOW_GENERIC_MESSAGE);
 	fwupd_request_set_message(
 	    request,
-	    "The update will continue when the device USB cable has been re-inserted.");
+	    "Wait 30 seconds after unplugging the device USB cable, the Phase 2 update will "
+	    "continue. Otherwise, it will continue to update the next time the USB cable is "
+	    "unplugged. The new FW will take effect automatically when the dock cable is "
+	    "disconnected from and connected to the computer again.");
```

Comment:
That's really hard to understand if you don't know english. Can we just ask the user to disconnect the cable, set a timer for 30 seconds and then ask the user to plug it back in?

Reply:
Sure, are you referring to the approach used in fu_wistron_dock_device_attach? I have a question: if the user does not follow the first instruction to unplug the cable and the instruction to plug in the cable still appears from the callback function, it seems that there is no longer a purpose to set the timer if the user goes on to unplug and plug the cable again?

Reply:
> fu_wistron_dock_device_attach

if we need to do that, yes. If I'm honest I'm really unimpressed with how the resets and physical actions are performed -- it seems really really fragile and easy to go wrong. We can't ask the user to wait 30 seconds -- user studies have shown that it's really hard for users to follow multi-stage processes with intervals.